File: python/867.py
def transpose(matrix: list[list[int]]) -> list[list[int]]:
    # new_matrix = []
    # for i in range(len(matrix[0])):
    #     temp = []
    #     for j in range(len(matrix)):
    #         temp.append(matrix[j][i])
    #     new_matrix.append(temp)
    # return new_matrix
    return [[matrix[j][i] for j in range(len(matrix))] for i in range(len(matrix[0]))]

File: javascript/867.js
var transpose = function (matrix) {
    let new_matrix = []
    for (let i = 0; i < matrix[0].length; i++) {
        let temp = []
        for (let j = 0; j < matrix.length; j++) {
            temp.push(matrix[j][i])
        }
        new_matrix.push(temp)
    }
    return new_matrix
};